chain/db/account.go

```diff
@@ -80,16 +173,20 @@ func (acc *account) SetNonce(nonce uint64) {
 	acc.nonce = nonce
 }
 
-func (acc *account) SetBalance(balance common.Fixed64) {
-	acc.balance = balance
+func (acc *account) SetBalance(assetID common.Uint256, symbol string, amount common.Fixed64) {
+	acc.balances[assetID] = &balance{
```

Comment:
I was thinking that, if `acc.balances[assetID]` already exists, should we just setting the value of each field in `acc.balances[assetID]` to avoid creating a new struct each time balance is changed?

Reply:
I will update this function.